soulstruct/base/project/editors/entities.py

```diff
@@ -692,7 +692,12 @@ def _write_entities_module(self):
 
         msb = self.get_selected_msb()
         try:
-            msb.write_entities_module(module_path, area_id=game_map.area_id, block_id=game_map.block_id)
+            msb.write_entities_module(
+                module_path,
+                area_id=game_map.area_id,
+                block_id=game_map.block_id,
+                include_vanilla_entities=self._project.include_vanila_entities
```

Comment:
The MSB class doesn't have access to the project configuration, which is why this boolean needs to be passed in. Luckily, editors have access to the project via the `_project` attribute.